just-install.json

```diff
@@ -398,6 +398,14 @@
       },
       "version": "5.0.4"
     },
+    "lockhunter": {
+      "installer": {
+        "kind": "innosetup",
+        "x86": "http://lockhunter.com/exe/lockhuntersetup_{{.version}}.exe",
+        "x86_64": "http://lockhunter.com/exe/lockhuntersetup_{{.version}}.exe"
```

Comment:
ConEmu actually does something similar. I prefer to add both entries only when URLs are different.